src/kudu/consensus/consensus_queue.cc

```diff
@@ -1204,8 +1211,7 @@ void PeerMessageQueue::AdvanceMajorityReplicatedWatermarkFlexiRaft(
   // 2 replicas in lla have received entries until index 5.
   std::map<std::string, std::vector<int64_t> > watermarks_by_region;
   for (const PeersMap::value_type& peer : peers_map_) {
-    if (replica_types == VOTER_REPLICAS &&
-        peer.second->peer_pb.member_type() != RaftPeerPB::VOTER) {
+    if (peer.second->peer_pb.member_type() != RaftPeerPB::VOTER) {
```

Comment:
is this change because we always pass in VOTER_REPLICAS?

Reply:
Yep